posts/2020-03-31-Rust-survey-2019.md

```diff
@@ -0,0 +1,216 @@
+
+In December 2018 we released the Rust 2018 edition - Rust 1.31.0. In the 2019 survey, 92% of Rust users indicated they were using the new edition. 85% said that upgrading to the Rust 2018 edition was easy. 
+
+![Comparative improvement](/images/2020-03-RustSurvey/18-Comparative-Improvement.png)
+
+Overall, many aspects of the Rust language were percieved as "somewhat better" in the 2018 edition.
+
+## Conferences and Community
+
+We noticed some differences between English language and other language results. Within the non-English language survey subset, the majority of the issues and concerns identified are the same as those within the English language. However, one concern/trend stands out among the non-English speaking subset - a desire for Rust documentation in their native language, or the language they took the survey in. This was particularly notable within the Chinese-language group, though that is likely due to the higher representation.
+
+We are tracking the [ongoing translation efforts](https://github.com/rust-lang/book/issues?q=is%3Aopen+is%3Aissue+label%3ATranslations) with the "Translation" GitHub issue label.
+
+We received a lot of feedback on how we can improve Rust and make it feel more welcoming to more people. We can't include all of it here, so here is a summary of some of the feedback that stood out to us.
+
+People are in general asking for more learning material about Rust. In terms of expertise it's mainly beginner and intermediate level material being requested. A lot of these requests also asked for video content specifically.
+
+The common blockers that people mention to participating is that they have social anxiety, and accessibility. One of the common reasons mentioned was that some resources are hard to read for people with dyslexia.
+
+These issues are definitely ones we want to address in 2020 and beyond.
+
+## Who is using Rust and what for?
+
+![How long have you worked in Rust](/images/2020-03-RustSurvey/7-How-Long-Worked-With-Rust.png)
+
+We found we have a lot of new folks coming in!
+
+![Size of summed Rust projects](/images/2020-03-RustSurvey/10-Size-Of-Summed-projects.png)
+
+The majority of Rust projects (43%) are 1,000-10,000 lines of code. Rust projects of medium to large size (those totaling over 10k and 100k lines of code respectively) continue to trend higher. They have grown from 8.9% in 2016, to 16% in 2017, to 23% in 2018, to **34%** in 2019.
+
+![How often do you use Rust](/images/2020-03-RustSurvey/11-How-Often-Use-Rust.png)
+
+Rust daily usage has trended slightly upward at 27.63% (it was just under 25% last year and 17.5% on 2017). Users who use Rust daily or weekly has also continued to rend slight upward. This year it was 68.52%, last year it was 66.4%, and in 2017 it was 60.8%.
+
+![How would you rate your Rust expertise](/images/2020-03-RustSurvey/12-How-Rate-Rust-Expertise.png)
+
+We also asked users how they would rate their Rust Expertise - there is a clear peak around "7".
```

Comment:
This point seems to be missing the elaboration on movement over prior years that we typically have; looking back here is the graph from 2018 and this years (respectively); it looks like this has on average not changed though we've had slightly less responses? I think noting the lack of change would be good, IMO that's a little concerning -- though as I think @nikomatsakis mentioned elsewhere, it would be nice to correlate this with "length of time using Rust" and "frequency of use". Hopefully, people are trending towards expert usage, especially if they use it weekly/daily.



![image](https://user-images.githubusercontent.com/5047365/78194576-8fabed80-744b-11ea-94f4-1970d1a33a35.png)

![image](https://user-images.githubusercontent.com/5047365/78194580-933f7480-744b-11ea-92cf-984d925457ed.png)